src/pages/Chaehyun/LoginBaeminSurvival.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import '../Chaehyun/LoginBaeminSurvival.scss';
+
+function LoginBaeminSurvival() {
+  const [inputId, setInputId] = useState('');
+
+  const [inputPw, setInputPw] = useState('');
+
+  const handleInputId = e => {
+    setInputId(e.target.value);
+  };
+
+  const handleInputPw = e => {
+    setInputPw(e.target.value);
+  };
+
+  const onSubmit = () => {
+    if (inputId === '') {
+      alert('아이디를 입력해주세요');
+    } else if (inputPw === '') {
+      alert('패스워드를 입력해주세요');
+    }
```

Comment:
```suggestion
    if (inputId === '') {
      alert('아이디를 입력해주세요');
      return;
    } 
    if (inputPw === '') {
      alert('패스워드를 입력해주세요');
      return;
    }
```
early return을 사용하면 좀 더 가독성 좋게 작성할 수 있습니다